examples/acados_matlab_octave/getting_started/simulink_example.m

```diff
@@ -26,5 +26,20 @@
 open_system(fullfile(target_folder, 'simulink_model_closed_loop'))
 
 
-%%
-disp('Press play in Simulink!');
+%% Run the models
+try
+    sim('simulink_model_integrator.slx');
+    cd ..
+catch
+    cd ..
+    error('Simulink integrator example failed')
+end
+
+try
+    cd c_generated_code
+    sim('simulink_model_closed_loop.slx');
+    cd ..
+catch
+    cd ..
+    error('Simulink closed loop example failed')
+end
```

Comment:
Does matlab crash otherwise?
I did a similar thing `examples/acados_matlab_octave/test; simulink_test`, which is run on github actions.

Reply:
The example works also without this modification, but I added it so it can be run from the testing script I added as well.